.github/SUPPORT.md

```diff
@@ -1,7 +1,7 @@
 # Support resources
 
 ## For HELICS and HELICS-FMI
-We have an active [Gitter]((https://gitter.im/GMLC-TDC/HELICS-src) channel
```

Comment:
How does Gitter interact with the name change?

Reply:
I don’t know yet. I’ve only seen an issue a few years ago asking them to move chat history from a room in one org to another

Reply:
Alright, I got an answer: https://gitlab.com/gitlab-org/gitter/webapp/blob/develop/docs/faq.md#what-happens-if-i-rename-something-on-github-org-repo

We could get the chat history migrated over to a room with the new name if we want to.